src/components/Overlay/Overlay.scss

```diff
@@ -0,0 +1,15 @@
+@import '~om';
+
+.Overlay {
+  @include stack(2);
```

Comment:
I think this is a slightly odd assumption to make (that everything in here should be vertically stacked with this spacing) — probably makes sense to have the consumer mixin a class that handles that but, I leave this to your judgement.